bundles/org.openhab.binding.sonnen/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -128,4 +143,16 @@
 		<description>Indicates if there is a current flow from production towards grid.</description>
 		<state readOnly="true"/>
 	</channel-type>
+	<channel-type id="kwhImportedState">
```

Comment:
Consider the more generic term "energy" rather than "kWh" since this channel type supports UoM.

Reply:
ok